spec/api_spec.rb

```diff
@@ -21,10 +21,10 @@ class Bar; end
       end
 
       context "when setting custom options" do
-        before { subject.setup foo: { bar: "baz" }, url: "https://api.example.com" }
+        before { subject.setup foo: { bar: "baz" }, url: "https://api.example.com", timeout: 10, open_timeout: 2 }
```

Comment:
You should be seeing a green build. The travis build passes on your [first commit](https://travis-ci.org/remiprev/her/builds/240389166?utm_source=github_status&utm_medium=notification) 😃